Week4/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/MovieListAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.e.blockbusterrecycler
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+
+
+class MovieListAdapter(
+    private val movies: List<ModelMovies>,
+    private val clickListener: MovieItemClicked
```

Comment:
Nice job passing in the list and the interface, this is a step that can be easilly missed.